src/io/ch5md.cpp

```diff
@@ -0,0 +1,431 @@
+/* This file is part of ch5md
+ * Copyright (C) 2013-2014 Pierre de Buyl
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the BSD license. See the LICENSE file for details.
```

Comment:
If this part is licensed under BSD (and not GPL-3 like the rest of Espresso) it should be noted in the `README` and `COPYING`. See https://www.softwarefreedom.org/resources/2007/gpl-non-gpl-collaboration.html